lib/run.js

```diff
@@ -87,6 +99,13 @@ function run (opts, cb) {
   url.setupClient = opts.setupClient
   url.timeout = opts.timeout
   url.requests = opts.requests
+  url.responseMax = opts.maxConnectionRequests
+
+  if (!url.responseMax) {
+    url.responseMax = typeof opts.maxOverallRequests === 'number'
+                      ? (opts.maxOverallRequests / opts.connections) : false
+    // this needs work if ((maxOverallRequests % opts.connections) > 0)
+  }
```

Comment:
You should add a comment on all of the above in the code, your sentence explain it, but otherwise it is not clear. I think that setting `responseMax` to `false` is generally a bad idea, because we say it is a number in the README.